src/CMakeLists.txt

```
@@ -139,49 +132,30 @@ set(SOURCES
     Timer.cpp
     ${CMAKE_CURRENT_BINARY_DIR}/git_commit_current.cpp)
 
-set(tgt __objects)
-add_library(${tgt} OBJECT ${SOURCES} ${INCLUDES})
-target_include_directories(${tgt} PRIVATE ${path_include_common})
-target_include_directories(${tgt} SYSTEM PRIVATE ${path_include_common_suppressed_warnings})
-
+add_library(forwarder-library ${SOURCES} ${INCLUDES})
```

See previous comment.
```suggestion
add_library(forwarder-library ${forwarder-library_SRC} ${forwarder-library_INC})
```
Also requires fixing of the `set(SOURCES`-line.